Whiskey/Tasks/New-WhiskeyProGetUniversalPackage.ps1

```diff
@@ -438,6 +287,17 @@ function New-WhiskeyProGetUniversalPackage
         Copy-ToPackage -Path $TaskParameter['ThirdPartyPath'] -AsThirdPartyItem
     }
 
+    $badChars = [IO.Path]::GetInvalidFileNameChars() | ForEach-Object { [regex]::Escape($_) }
+    $fixRegex = '[{0}]' -f ($badChars -join '')
+    $fileName = '{0}.{1}.upack' -f $name,($version.SemVer2NoBuildMetadata -replace $fixRegex,'-')
+
+    $outFile = Join-Path -Path $TaskContext.OutputDirectory -ChildPath $fileName
+
+    Write-WhiskeyVerbose -Context $TaskContext -Message $fileName

```

Comment:
I think this is a good idea but too much information. Write a verbose statement before each call to `Invoke-WhiskeyRobocopy` that shows the source path being copied and the relative path in the package it is being copied to.

Reply:
That is already being written with `Write-WhiskeyInfo` on [Line 255](https://github.com/webmd-health-services/Whiskey/blob/87c4936349ba4149f63abd8088e3bf8d0bdadebe/Whiskey/Tasks/New-WhiskeyProGetUniversalPackage.ps1#L255). Should I change it to `Write-WhiskeyVerbose` instead?

I'll remove the `Tree` command output.

Reply:
No. Forgot about that Write-WhiskeyInfo. Just remoe the `tree` command output.